src/MAIN_NEMS.F90

```diff
@@ -66,6 +66,14 @@ PROGRAM MAIN_NEMS
        USE module_NEMS_Rusage,ONLY: NEMS_Rusage
 !
 !-----------------------------------------------------------------------
+!***  This module contains PIO initialization related routines.
+!-----------------------------------------------------------------------
+!
+#if defined PIO
```

Comment:
Does this mean the PIO used here is from the PIO module in the module configuration file? If I remember correctly, we also have pio inside CMEPS.

Reply:
The PIO macro used by the preprocessor wraps the code used to initialize PIO in the NEMS driver. When compiling with -DPIO then the modules must include shr_pio_mod and libraries must include shr_pio_init1 (NEMS driver) and shr_pio_init2 (NEMS program). The modulefile and CMake build system will need to link in the correct libraries/modules.  @uturuncoglu can provide more information.

Reply:
Let us know if you think -DPIO isn't the best macro name to include PIO initialization within the NEMS driver.

Reply:
I am just curious, I thought PIO is already used in the model (inside CMEPS), so do we need to use -DPIO?

Reply:
@junwang-noaa PIO preprocessor flag is not used by CMEPS. There is an INTERNAL_PIO_INIT preprocessor to activate PIO initialization in CMEPS side but this is not used anymore since we have two PIO dependent component in the UFS model. If you think that using -DPIO confuses the developers/users we could use PIO_INIT etc. but this also requires change in the model level CmakeLists.txt file.

Reply:
I will mark this as resolved currently. But keep in mind, we will need address or unify this when we try to merge the support/HAFS branch back to NEMS's develop branch.

Reply:
@BinLiu-NOAA sure. maybe opening an issue/reminder for it could be a good idea.